### PR TITLE
draco: add v7.19.0

### PR DESCRIPTION
**Pull request by clevelam**
Update to latest Draco-7_19_0 version

### CODE REVIEW

**Comment by clevelam** [comment]
@keadyk can you review this for me?

**Comment by KineticTheory** [comment]
@clevelam 

While testing this build I ran into this error:

```shell
  >> 85    CMake Error at config/unix-g++.cmake:12 (message):
     86      Draco requires GNU gcc/g++ version >= 9.0.
```

Can you add the following command to the `conflicts` block to catch this issue earlier?

```python
# Newer versions of Draco require newer compilers.
with when("@7.19.0:"):
    conflicts("%gcc@:9.0")
```

This will allow spack to fail with:

```
==> Error: failed to concretize `draco@7.19.0` for the following reasons:
     1. draco: '%gcc@:9.0' conflicts with '@7.19.0:'
        required because conflict is triggered when %gcc@:9.0
```

**Review by KineticTheory — CHANGES_REQUESTED** [review]
See comments I left on the main discussion page. https://github.com/spack/spack/pull/47032#issuecomment-2418105315

**Review by bernhardkaindl — DISMISSED** [review]
Auto-submitted build results on Ubuntu 22.04.4 LTS: (see disclaimer below)
The following specs were checked (extracted from `gh pr diff`):
- `draco@7.19.0`

The following specs were installed or passed building in this run: (variants are shortened, e.g. no disabled variants)
```py
bin/spack find --variants draco@7.19.0 | sed 's/~[a-z]*//g;s/ [a-z0-9_]*=[a-zA-Z0-9]*//g'-- linux-ubuntu22.04-skylake / gcc@13.1.0 -----------------------
draco@7.19.0+eospac_fma+lapack+libquo+parmetis+superlu_dist
```
Generated and submitted by https://github.com/spack/build-quality-tools/blob/main/build_pr_changes.py
```py
~/build-quality-tools/build_pr_changes.py -q new-packages.txt -may
```
Thanks to @KineticTheory for the review, I need to improve my script for reviews by maintainers!

**Comment by clevelam** [comment]
> @clevelam
> 
> While testing this build I ran into this error:
> 
> ```shell
>   >> 85    CMake Error at config/unix-g++.cmake:12 (message):
>      86      Draco requires GNU gcc/g++ version >= 9.0.
> ```
> 
> Can you add the following command to the `conflicts` block to catch this issue earlier?
> 
> ```python
> # Newer versions of Draco require newer compilers.
> with when("@7.19.0:"):
>     conflicts("%gcc@:9.0")
> ```
> 
> This will allow spack to fail with:
> 
> ```
> ==> Error: failed to concretize `draco@7.19.0` for the following reasons:
>      1. draco: '%gcc@:9.0' conflicts with '@7.19.0:'
>         required because conflict is triggered when %gcc@:9.0
> ```

@KineticTheory can you try this updated change set for me?

File: var/spack/repos/builtin/packages/draco/package.py
Comment (KineticTheory) [inline var/spack/repos/builtin/packages/draco/package.py]:
```suggestion
    with when("@7.19.0:"):
```

**Review by KineticTheory — APPROVED** [review]
LGTM

**Review by bernhardkaindl — COMMENTED** [review]
Build results on Ubuntu 22.04.4 LTS: These changed specs were found `gh pr diff`:
- `draco@7.19.0`

These specs were installed:
```py
bin/spack find --deps --variants draco@7.19.0
```
### linux-ubuntu22.04-skylake / gcc@13.1.0:
<details><summary>Click on this line to unfold and refold the dependencies of this build


```py
draco@7.19.0+eospac_fma+lapack+libquo+parmetis+superlu_dist build_system=cmake
```
</summary>

Dependency spec tree (disabled variants are removed in this output):
```py
    cmake@3.30.5+ncurses+ownlibs build_system=generic patches=dbc3892
        curl@8.8.0+nghttp2 libs=shared,static tls=openssl
            nghttp2@1.63.0
        ncurses@6.5+termlib abi=none patches=7a351bc
        zlib-ng@2.2.1+compat+new_strategies+opt+pic+shared
    eospac@6.5.11 build_system=generic
    gcc-runtime@13.1.0 build_system=generic
    glibc@2.35
    gmake@4.4.1 build_system=generic
    gsl@2.7.1-cblas+pic+shared
    libquo@1.4
    metis@5.1.0+shared build_system=cmake patches=4991da9,93a7903,b1225da
    numdiff@5.9.0
    openblas@0.3.28_fpcsr+dynamic_dispatch+fortran+locking+pic+shared build_system=makefile patches=d0b9276 symbol_suffix=none threads=none
    openmpi@5.0.5+atomics-hwloc-libevent-pmix+rsh+vt+wrapper-rpath fabrics=none romio-filesystem=none schedulers=none
        autoconf@2.72
            m4@1.4.19+sigsegv patches=9dc5fbd,bfdffa7
                libsigsegv@2.14
        automake@1.16.5
        hwloc@2.11.1+libxml2-level-zero+pci libs=shared,static
            libpciaccess@0.17
                util-macros@1.20.1
            libxml2@2.10.3+pic+shared
        libevent@2.1.12+openssl
        libtool@2.4.7
            findutils@4.9.0 patches=440b954
        numactl@2.0.18
        openssh@9.8p1+gssapi
            krb5@1.21.2+shared
                bison@3.8.2
            libedit@3.1-20240808
            libxcrypt@4.4.35_api patches=4885da3
        perl@5.40.0+cpanm+opcode+open+shared+threads build_system=generic
            berkeley-db@18.1.40+cxx+stl patches=26090f4,b231fcc
        pkgconf@2.2.0
        pmix@5.0.3
    parmetis@4.0.3+shared build_system=cmake patches=4f89253,50ed208,704b84f
    python@3.13.0+bz2+ctypes+dbm+libxml2+lzma+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl+uuid+zlib build_system=generic
        bzip2@1.0.8+shared build_system=generic
            diffutils@3.10
        expat@2.6.3+libbsd
            libbsd@0.12.2
                libmd@1.0.4
        gdbm@1.23
        gettext@0.22.5+bzip2+curses+git+libxml2+pic+shared+tar+xz
            tar@1.34 zip=pigz
                pigz@2.8 build_system=makefile
                zstd@1.5.6+programs build_system=makefile compression=none libs=shared,static
        libffi@3.4.6
        openssl@3.3.1+shared build_system=generic certs=mozilla
            ca-certificates-mozilla@2023-05-30 build_system=generic
        readline@8.2 patches=bbf97f1
        sqlite@3.46.0+column_metadata+dynamic_extensions+fts+rtree
        util-linux-uuid@2.40.2
        xz@5.4.6 libs=shared,static
    random123@1.14.0 build_system=generic patches=2d69277
```
</details>

Generated by https://github.com/spack/build-quality-tools/blob/main/build_pr_changes.py -am

**Review by bernhardkaindl — APPROVED** [review]

**Pull request merged**
